### PR TITLE
dump the JSON files at the end of the day

### LINKED ISSUE #37: Post daily JSON objects of user-inputted data to Dropbox
Once the user is done for a day: we should upload a file of what they inputted in the episode creation and prediction screen as a CSV up to dropbox.

### PR DESCRIPTION
**Pull request by dfarooq610**
Writes JSON files for each day after they confirm on the episode prediction screen. 

To Do: On day 3 we are going to have to upload the JSON files, but there is no episode prediction screen, we gotta upload the info somehow. When we get there, use that helper.

### CODE REVIEW

**Review by zefwang — APPROVED** [review]
LGTM, glad you were able to figure out the upload format stuff.

File: utils/AsyncStoageUtils.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
 export const STORAGE_KEYS = {
-  start_day: () => "StartDay",
+  startDay: () => "StartDay",
   daysEpisodes: (recordingDay: number) => `Day${recordingDay}Episodes`,
   episodeRecall: () => "EpisodeRecall",
+  episodePredictions: (recordingDay: number) =>
```
Comment (dfarooq610) [inline utils/AsyncStoageUtils.ts]: I am gonna revert this since we are never going to actually store episode predictions in storage

Comment (imanmoreira) [reply utils/AsyncStoageUtils.ts]: Just a little bump to remember to do this

**Review by imanmoreira — APPROVED** [review]
Looks good!

**Pull request merged**